README.md

```diff
@@ -109,26 +109,193 @@ which appears to overlap with a subset of `OpenAssetIO`s concerns.
+then execute the tests.
+
+A disadvantage of this is that the pure Python component is
+reinstalled every time tests are executed, even if no files changed.
+
+#### Step by step
+
+We must first build and install the binary artifacts. Assuming the CMake
+build has been configured as above, then
+
+```shell
+cmake --build build
+cmake --install build
+```
+
+Since the discovered Python may be different from the system Python,
+there is a convenience build target to create a Python virtual
+environment in the installation directory, which ensures that the same
+Python that was linked against is used to create the environment.
+
+```shell
+cmake --build build --target openassetio-python-venv
+```
+
+We can then use this environment to execute the tests.
+
+Next we must install the pure Python component and test-specific
+dependencies. Assuming the install directory is `dist` under the
+repository root (the default)
+
+```shell
+source dist/bin/activate
+pip install .
 pip install -r tests/requirements.txt
-pytest
 ```
 
+We can now run the Python tests via `pytest`
+
+```shell
+pytest tests
+```
+
+### Using Vagrant
+
+Included for convenience is a [Vagrant](https://www.vagrantup.com/)
+configuration for creating reproducible build environments. This is
+configured to create a virtual machine that matches the Linux Github CI
+environment as close as is feasible.
+
+The Vagrant VM installs third-party library dependencies using the
+[conan](https://conan.io/) package manager, and sources the dependencies
+from the default public [ConanCenter](https://conan.io/center/)
+repository.
+
+In order to build and run the tests within a Vagrant VM, assuming
+Vagrant is installed and the current working directory is the root of
+the repository, run the following
+
+```shell
+cd ci
+vagrant up
+# Wait a while...
+vagrant ssh
+cmake -S openassetio -B build --install-prefix ~/dist \
+  --toolchain ~/conan/conan_paths.cmake -DOPENASSETIO_ENABLE_TESTS=ON
+```
+
+Then we can run the following steps to build, install and run the tests
+
+```shell
+cmake --build build
+cmake --install build
+cmake --build build --target openassetio-python-venv
+source dist/bin/activate
+pip install ./openassetio
+pip install -r openassetio/tests/requirements.txt
+pytest openassetio/tests
+```
+or alternatively, simply
+
+```shell
+cd build
+ctest
+```
+
+### Using the ASWF Docker image
+
+The Academy Software Foundation maintains a set of VFX Reference 
+Platform compatible [Docker](https://www.docker.com/) images [here](https://github.com/AcademySoftwareFoundation/aswf-docker).
+
+The CY22 image contains all dependencies currently required for building
+OpenAssetIO. For example, to build and run the tests via a container, 
+from the repository root run
+
+```shell
+docker run -v `pwd`:/src aswf/ci-base:2022.2 bash -c '
+  cd /src && cmake -S . -B build -DOPENASSETIO_ENABLE_TESTS=ON &&\
+  cd build && ctest -VV'
+````
```

Comment:
[fix] Stray extra backtick (sorry I couldn't figure the escape syntax to make it a suggestion)

Reply:
Fixed as part of 5212172